docs/reference-guides/data/data-core.md

```diff
@@ -150,6 +150,10 @@ _Returns_
 
 -   `undefined< 'edit' >`: Current user object.
 
+### getDefaultTemplateId
```

Comment:
This will be extracted to its own PR, it just replaces the adhoc template lookup calls with proper core-data selector/resolver.